new_core/screens/ingame.js

```diff
@@ -136,10 +144,23 @@ export function update () {
     moveScreen(BOUNDARY_Y, shiftY)
   }
 
-  if (shiftX || shiftY) {
-    prepareTween(inputDelay, { step: 0 })
+
+  if (!somethingIsSelected && input.ACTION) {
+    cursor.selected.unit = null
+    cursor.selected.property = null
+
+    const unitIdAtPosition = map.tiles[cursor.x][cursor.y].unitId;
+    if (unitIdAtPosition !== -1) {
+      cursor.selected.unit = units[unitIdAtPosition]
+    }
   }
 
+  updateTween(animationData, true, loop.delta)
+
+  //if (shiftX || shiftY) {
```

Comment:
I commented this out, it was from the master code, but I was unsure if it'd break something if I got rid of it completely. Next huge commit, this will most likely be cleaned up.